Control/Monad/Logger.hs

```diff
@@ -116,6 +116,9 @@ import Control.Concurrent.STM.TBChan
 import Control.Exception.Lifted (onException, bracket)
 import Control.Monad (liftM, ap, when, void, forever)
 import Control.Monad.Base (MonadBase (liftBase), liftBaseDefault)
+#if __GLASGOW_HASKELL__ >= 801
```

Comment:
This is more typically done with `MIN_VERSION_base` macros. Can you switch over to use those?

Reply:
Fixed using rebase.